### PR TITLE
Tracing chrome span names

### PR DESCRIPTION
**Pull request by superdump**
This should work in theory but for some reason the tracing-chrome profile data is not being properly flushed to the .json file it creates. I wonder if the thread-local resource isn't doing what we want it to?

### CODE REVIEW

File: crates/bevy_log/src/lib.rs [hunk 1]
```diff
@@ -12,7 +12,12 @@ pub use bevy_utils::tracing::{
 };
 
 use bevy_app::{AppBuilder, Plugin};
-use tracing_subscriber::{prelude::*, registry::Registry, EnvFilter};
+use tracing_subscriber::{
+    fmt::{format::DefaultFields, FormattedFields},
```
Comment (amberkowalski) [inline crates/bevy_log/src/lib.rs:16]: These are marked unused by `cargo check`, causing CI to fail ):

**Comment by cart** [comment]
Works for me!

```
cargo run --example 3d_scene --release --features trace,trace_chrome
```

image

File: crates/bevy_log/src/lib.rs (hunk 1 above)
Comment (superdump) [reply crates/bevy_log/src/lib.rs]: Fixed. Thanks.

**Comment by superdump** [comment]
> Works for me!
> 
> ```
> cargo run --example 3d_scene --release --features trace,trace_chrome
> ```
> 
> image

I forgot to enable the trace feature. Works for me too now. Yey!

**Pull request merged**